### PR TITLE
Define a node & assesssment for reactivity

### PR DESCRIPTION
**Pull request by sskutnik**

### CODE REVIEW

File: define-reactivity/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+uuid: define-reactivity
+layout: node
```
Comment (TheDoctorRAB) [inline define-reactivity/index.md]: layout should be first

File: n-batch-core/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+uid: n-batch-cores
```
Comment (TheDoctorRAB) [inline n-batch-core/index.md]: uuid

File: n-batch-core/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+layout: node
```
Comment (TheDoctorRAB) [inline n-batch-core/index.md]: layout first

**Review by TheDoctorRAB — CHANGES_REQUESTED** [review]
layout first
uuid not uid

**Comment by sskutnik** [comment]
@TheDoctorRAB Ordering fixed.

**Pull request merged**